jetty-websocket/jetty-websocket-tests/src/test/java/org/eclipse/jetty/websocket/tests/EchoCreator.java

```diff
@@ -18,14 +18,14 @@
 
 package org.eclipse.jetty.websocket.tests;
 
-import org.eclipse.jetty.websocket.servlet.ServletUpgradeRequest;
-import org.eclipse.jetty.websocket.servlet.ServletUpgradeResponse;
-import org.eclipse.jetty.websocket.servlet.WebSocketCreator;
+import org.eclipse.jetty.websocket.server.JettyServerUpgradeRequest;
+import org.eclipse.jetty.websocket.server.JettyServerUpgradeResponse;
+import org.eclipse.jetty.websocket.server.JettyWebSocketCreator;
```

Comment:
I know the `Jetty` prefix makes these classes much easier for us to understand during development.... but I'm wondering if for our users it would be better to keep them named as `ServerUpgradeRequest`,  `ServerUpgradeResponse`, `WebSocketCreator`, so that our users only need to change the import from `servlet` to `server` and make no more code changes?